timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/common/ElasticsearchUtil.java

```diff
@@ -346,6 +354,29 @@ public static void drainAndIndex(BlockingQueue<Event> eventsQueue, TaskIndexer t
 		}
 	}
 
+	private static Collection<Event> filterEvents(Collection<Event> unfilteredEvents, String skipEventsAtDrainFlag, String notToSkipRegex) {
+		if (Boolean.parseBoolean(skipEventsAtDrainFlag)) {
+			return unfilteredEvents.stream()
+					.filter(event-> shouldKeep(event, notToSkipRegex))
```

Comment:
I think you should have a log here for any event you filter and any event that isn't filtered out